lib/publishing_api_payload/file_attachment.rb

```diff
@@ -0,0 +1,22 @@
+class PublishingApiPayload::FileAttachment
+  include Rails.application.routes.url_helpers
+  include FileAttachmentHelper
+
+  attr_reader :attachment, :document
+
+  def initialize(attachment, document)
+    @attachment = attachment
+    @document = document
+  end
+
+  def payload
+    payload = {
+      attachment_type: "file",
+      locale: document.locale,
+      preview_url: file_attachment_preview_url(attachment, document),
```

Comment:
In Publishing API metadata this field is intended to be the public path to a CSV preview. So we shouldn't have a CP link. We can not fill this field for the time being.